api/serverless.ts

```diff
@@ -293,19 +293,24 @@ serverlessConfiguration.functions = {
         }
       : undefined
   ),
-  healthCheck: healthCheck(
-    env.CI
-      ? {
-          securityGroupIds: ["${self:custom.config.infrastructure.SECURITY_GROUP}"],
-          subnetIds: [
-            "${self:custom.config.infrastructure.SUBNET_01}",
-            "${self:custom.config.infrastructure.SUBNET_02}",
-          ],
-        }
-      : undefined
-  ),
 };
 
+if (stage !== "content" && stage !== "testing") {
```

Comment:
Longer term thought: I know these are the right names, and I checked `config.yml` to confirm. It does make me think that we should have variables for these just to potentially avoid typos in this file.

Reply:
Added variables to resolve this.